src/init/bitcoin-qt.cpp

```diff
@@ -16,6 +16,8 @@
 
 namespace init {
 namespace {
+const char* EXE_NAME = "bitcoin-node";
```

Comment:
231fba1f067211b889080bd97c8f58df1500ecf1: I'm confused, should this be `bitcoin-qt`?

I don't see it appear when I run `build/bin/bitcoin gui --version`